README.md

```diff
@@ -1 +1,3 @@
-Este va a ser el archivo README de mi aplicacion de prueba con Flask.
+ To use an emulated email server, Python provides one that is very handy that you can start in a
+ second terminal with the following command: "$ python -m smtpd -n -c DebuggingServer localhost:8025"
```

Comment:
You can wrap code snipets in ` for single line or ``` for multiline so they look better

Reply:
It will be fixed in the next PR in the next branch.